_episodes/07-vector-data-in-python.md

```diff
@@ -149,126 +170,189 @@ array([109222.03325 , 469461.512625, 140295.122125, 510939.997875])
+
+ ```
+ # A smaller bounding box in RD
+ xmin, xmax = (120_000, 135_000)
+ ymin, ymax = (485_000, 500_000)
 
-![Extent image](../fig/E07-01-spatial_extent.png)
+ fields_cx = fields.cx[xmin:xmax, ymin:ymax]
+ ```
+
+## Export data to file
+
+We will save the cropped results to a shapefile (`.shp`) and use it later. The `to_file` function can be used for exportation:
+
+~~~
+fields_cx.to_file('fields_cropped.shp')
+~~~
+{: .language-python}
+
+
+This will write it to disk (in this case, in 'shapefile' format), containing only the data from our cropped area. It can be read again at a later time using the `read_file()` method we have been using above. Note that this actually writes multiple files to disk (`fields_cropped.cpg`, `fields_cropped.dbf`, `fields_cropped.prj`, `fields_cropped.shp`, `fields_cropped.shx`). All these files should ideally be present in order to re-read the dataset later, although only the `.shp`, `.shx`, and `.dbf` files are mandatory (see the [Introduction to Vector Data]({{site.baseurl}}/02-intro-to-vector-data) lesson for more information.)
 
-We can convert these coordinates to a bounding box or acquire the index of the dataframe to access the geometry. Either of these polygons can be used to clip rasters (more on that later).
 
 ## Selecting spatial features
-Sometimes, the loaded data can still be too large. We can cut it is to a even smaller extent using the `.cx` indexer (note the use of square brackets instead of round brackets, which are used instead with functions and methods):
+
+From now on, we will take in a point dataset `brogmwvolledigeset.zip`, which is the underground water monitoring wells. We will perform vector processing on this dataset, together with the cropped field polygons `fields_cropped.shp`.
+
+Let's read the two datasets.
 
 ~~~
-# Define a Boundingbox in RD
-xmin, xmax = (120_000, 135_000)
-ymin, ymax = (485_000, 500_000)
-cropfield_crop = cropfield.cx[xmin:xmax, ymin:ymax]
+fields = gpd.read_file("fields_cropped.shp")
+wells = gpd.read_file("data/brogmwvolledigeset.zip")
 ~~~
 {: .language-python}
 
+And take a look at the wells:
 
-This will cut out a smaller area, defined by a box in units of the projection, discarding the rest of the data. The resultant GeoDataframe, which includes all the features intersecting the box, is found in the `cropfield_crop` object. Note that the edge elements are not 'cropped' themselves. We can check the total bounds of this new data as before:
+~~~
+wells.plot(markersize=0.1)
+~~~
+{: .language-python}
+![All wells](../fig/E07-03-wells-nl.png)
+
+The points represents all the wells over the Netherlands. Since the wells are in the lat/lon coordinates. To make it comparable with fields, we need to transfer the CRS to RD first:
```

Comment:
```suggestion
The points represents all the wells over the Netherlands. Since the wells are in the lat/lon coordinates. To make it comparable with fields, we need to first transfer the CRS to the "RD New" projection:
```